src/styles/mixins.less

```diff
@@ -245,25 +245,33 @@
   box-shadow: 0 0 0 @edu-outline-width fade(@edu-error-color, @edu-outline-fade);
 }
 
-.m-dropzone-transition {
-  transition:
-    background-color @edu-animation-duration-slow,
-    border-color @edu-animation-duration-slow,
-    box-shadow @edu-animation-duration-slow;
-}
 
-.m-dropzone-base {
-  .m-dropzone-transition;
-  background-color: @edu-background-color-light;
-  border: 1px dashed @edu-border-color-active;
+.m-dropzone {
+  position: relative;
   border-radius: @edu-border-radius-base;
+  border: 1px dashed @edu-border-color-active;
+  background-color: @edu-background-color-light;
 }
 
 .m-dropzone-active {
-  .m-dropzone-transition;
-  background-color: @edu-background-color-base;
   border-style: solid;
+  border-radius: @edu-border-radius-base;
   .m-border-glow-success;
+
+  &::after {
+    content: '';
+    .m-mask;
```

Comment:
There is a tiny glitch in case the element scrolls (as it does for example in `MediaLibrarySearchScreen-noSearch.is-dropping` on mobile screens): The mask scrolls together with the element content (making the mask disappear at some point when scrolling down). It is not very visible since we use it on a light gray background, and - according to my research - there is no way to fix this using `::before`/`::after`. I could live with this, just weanted to point out. After all, the mask is just an additional visual indication, there's also the the outer glow.